bin/hammer

```diff
@@ -42,6 +42,13 @@ end
 # load user's settings
 require 'hammer_cli/settings'
 
+hammer_gems = Gem::Specification.select do |spec|
+  spec.name.include?('hammer_cli')
+end
+CFG_PATH_GEM = hammer_gems.collect { |spec| "#{spec.gem_dir}/config" }
+
+HammerCLI::Settings.load_from_paths CFG_PATH_GEM
+
 CFG_PATH_LEGACY = ['~/.foreman/', '/etc/foreman/', "#{::RbConfig::CONFIG['sysconfdir']}/foreman/"].uniq
 HammerCLI::Settings.load_from_paths CFG_PATH_LEGACY
 if HammerCLI::Settings.path_history.empty?
```

Comment:
This condition causes failures for me. When there are some default configs discovered and loaded, the path is recorded in the path history. The condition was there to check for legacy config files. As a result the user's configuration is skipped:
```
hammer 
Warning: Legacy config paths detected, move the following files
    /root/hammer/hammer-cli-foreman/config/foreman.yml -> /root/hammer/hammer-cli-hammer/config/foreman.yml
    /root/hammer/hammer-cli/config/cli_config.template.yml -> /root/hammer/hammer-cli/config/cli_config.template.yml
Could not load the API description from the server
  - is the server down?
  - was 'foreman-rake apipie:cache' run on the server when using apipie cache? (typical production settings)
Warning: An error occured while loading module hammer_cli_foreman
```

I think we can remove the legacy code now. It's been there for long enough: https://github.com/theforeman/hammer-cli/commit/9959a4fb145d1970cc1e70abba0023d177b82a6f